database/data.sql

```diff
@@ -0,0 +1,25 @@
+--Sample data
+insert into "availabilityTypes" ("label")
+     values ('Weekends Morning'),
+            ('Weekends Afternoon'),
+            ('Weekends Evening'),
+            ('Weekdays Morning'),
+            ('Weekdays Afternoon'),
+            ('Weekdays Evening');
+
+insert into "Users" ("firstName", "lastName", "email", "password", "profileImage", "bio", "interests")
+     values ('Helen', 'Biren', 'helenbiren@aol.com', 'hash789', 'server/public/images/user1.png', 'Born in Redhook, Brooklyn, in the year of who knows when. Opened up her eyes to the tune of an accordion.', 'skiing, running, healthy food, hiking, movies, music'),
+            ('Kristy', 'San', 'ksan@mac.com', 'dsa432', 'server/public/images/kristy.png', 'I enjoy food and going to different restaurants.', 'food, Italian food, cuisine, hiking');
+
+insert into "activityTypes" ("label")
+     values ('Food'),
+            ('Sports'),
+            ('Museum');
+
+insert into "Messages" ("messageContent")
+     values ('Hello. How are you?'),
+            ('Good and you?'),
+            ('Fine.');
+
+insert into "Activities" ("googlePlacesLink", "googleMapsLink", "specificActivity", "location", "date", "time")
+     values ('https://maps.googleapis.com/maps/api/place/textsearch/json?query=salad+restaurants+in+koreatown+los+angles+California&key=${process.env.GOOGLE_PLACES_API_KEY}', 'https://www.google.com/maps/embed/v1/place?key=${process.env.GOOGLE_PLACES_API_KEY}', 'restaurant', 'sweetgreen', '01/01/2021', '1PM');
```

Comment:
You should only add your API key right when you send the request, and not store it. this will be problematic if you ever need to update your key.

Reply:
The `URL` and `URLSearchParams` classes make it pretty easy to manipulate url strings.